cmd/helm/installer/options.go

```diff
@@ -46,6 +46,9 @@ type Options struct {
 	// ServiceAccount is the Kubernetes service account to add to Tiller.
 	ServiceAccount string
 
+	// Force allows to force upgrading tiller if deployed version is greater than current version
+	ForceUpgrade bool
```

Comment:
We can potentially rename to "Force" although I don't really see any other use for it. Found Personally I found ForceUpdate more clear while navigating